evmrpc/rpcstack.go

```diff
@@ -299,6 +299,7 @@ func (h *HTTPServer) EnableRPC(apis []rpc.API, config HTTPConfig) error {
 	if err := RegisterApis(h.log, apis, config.Modules, srv); err != nil {
 		return err
 	}
+	h.log.Info(fmt.Sprintf("[Debug] Registering deny list for evm rpc:%v", config.DenyList))
```

Comment:
do we want to leave this as debug or just make it a regular log?

Reply:
Should be fine to leave it since it's an one-off log during startup